pkg/lightning/backend/local/region_job.go

```diff
@@ -436,6 +436,10 @@ func (local *Backend) doWrite(ctx context.Context, j *regionJob) error {
 		allPeers = append(allPeers, peer)
 	}
 	dataCommitTS := j.ingestData.GetTS()
+	if dataCommitTS == 0 {
```

Comment:
for `intest` maybe add a range test that TS should be in range `[now-1m, now)`

Reply:
In real world execution, the TS is determined and persisted in subtask metadata, maybe it's earlier than now-1m. When we test failover, chaos test, or wjhuang2016's SQL fuzzing, intest flag may be turned on and become flaky tests

Reply:
anyway, we need some basic range check, either 1m, 1h, or even 1d